vscode/README.md

```diff
@@ -4,15 +4,15 @@
 [![Visual Studio Marketplace Installs](https://img.shields.io/visual-studio-marketplace/i/toit.toit.svg?style=flat-square)](https://marketplace.visualstudio.com/items?itemName=toit.toit)
 [![Visual Studio Marketplace Downloads](https://img.shields.io/visual-studio-marketplace/d/toit.toit.svg?style=flat-square)](https://marketplace.visualstudio.com/items?itemName=toit.toit)
 
-The Toit extension add language support for the Toit programming language, including syntax highlighting, integrated static analysis, code completion, and device overview.
+The Toit extension adds language support for the Toit programming language, including syntax highlighting, integrated static analysis, code completion, and device overview.
 
 ## Quick start
 
-- *Step 1:* Install [Toit](https://docs.toit.io/getstarted/installation) and the [toit Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=toit.toit).
+- *Step 1:* Install [Toit](https://docs.toit.io/getstarted/installation) and the [Toit Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=toit.toit).
 - *Step 2:* Either place `toit` in your path or set `toit.Path` to the location of `toit`.
 - *Step 3:* The extension is activated when you open a `.toit` file.
 
-The extension is now setup and ready to use. Visit the [Toit docs](https://docs.toit.io/) for more information about Toit.
+The extension is now set up and ready to use. Visit the [Toit docs](https://docs.toit.io/) for more information about Toit.
```

Comment:
AFAIK, setup is an adjective here and should thus be one word.

Reply:
I don't think that is true, but I could be wrong. @erikcorry to the rescue? :)

Reply:
If it was a noun it would be one word.  If it was a verb it should be two.  Here it is an adjectival verb, and should be two words.